mne_features/tests/test_univariate.py

```diff
@@ -521,9 +521,9 @@ def test_feature_names_energy_freq_bands():
     fb1 = np.array([[4., 8.], [30., 70.]])
     fb2 = {'theta': [4, 8], 'low-gamma': np.array([30, 70])}
     _fb = [fb1, fb2]
-    expected_names1 = ['ch0_band0', 'ch0_band1', 'ch1_band0', 'ch1_band1']
-    expected_names2 = ['ch0_theta', 'ch0_low-gamma',
-                       'ch1_theta', 'ch1_low-gamma']
+    expected_names1 = ['ch0__band0', 'ch0__band1', 'ch1__band0', 'ch1__band1']
+    expected_names2 = ['ch0__theta', 'ch0__low-gamma',
+                       'ch1__theta', 'ch1__low-gamma']
```

Comment:
you are changing the behavior here without deprecation... it's not ideal even though I don't think this package has hundreds of users.

Reply:
I have updated the code so the changes only applied to the `FeaturesExtractor` output. Behavior stays the same for all feature names. Only the introduced feature names in `FeaturesExtractor` now follow the format `<ch_name>__<func_params>__<feature_name>`.